README.md

```diff
@@ -1,84 +1,104 @@
 # preCICE-adapter for the CFD code ANSYS Fluent
-Developed by Bernhard Gatzhammer Update-Try to newer Fluent and Precice Versions
-
-1. How to build the Fluent-preCICE adapter: 
-* Put the library of preCICE (libprecice.so) into the lnamd64 folder.
-* Adapt lnamd64/2ddp_host/user.udf line 1 "CSOURCES=...": There are several main udf files
-    - fsi_udf.c: For FSI simulations. Needs fsi.c.
-    - wave_profile_udf.c: For wave simulations with inflow profile. Needs 
-      wave_profile.c.
-    - wave_maker_udf.c: For wave simulations with moving wall. Needs 
-      wave_maker.c.
-    - fsi_and_wave_profile_udf.c: For FSI + wave simulations. Needs fsi.c and 
-      wave_profile.c.
-    The variable SOURCES needs one one main udf file and the correspoding .c files.
-    
-* Adapt the path of the python library in /src/makefile line 19
-* Type: make "FLUENT_ARCH=lnamd64" to start the build. Add a "clean" to clean it.
+*Developed by Bernhard Gatzhammer and valid for FLUENT 19.1 and preCICE v1.4*
+
+## 1. How to build the Fluent-preCICE adapter: 
+  * Put the library of preCICE (libprecice.so) into the lnamd64 folder.
+  * Adapt lnamd64/2ddp_host/user.udf line 1 "CSOURCES=...": There are several main udf files
+      The variable SOURCES needs one main udf file and the correspoding .c files.
+      - fsi_udf.c: For FSI simulations. Needs fsi.c.
+      - wave_profile_udf.c: For wave simulations with inflow profile. Needs 
+        wave_profile.c.
+      - wave_maker_udf.c: For wave simulations with moving wall. Needs 
+        wave_maker.c.
+      - fsi_and_wave_profile_udf.c: For FSI + wave simulations. Needs fsi.c and 
+        wave_profile.c.
+
+  * Adapt lnamd64/2ddp_host/user.udf line 3 "FLUENT_INC = ..." to the fluent installation
+    folder which will be of the type *./ansys_inc/v191/fluent*  
+  * Adapt the path of the python library in /src/makefile line 19
+  * Update the Ansys RELEASE version in /src/makefile line 26
+  * Type: make "FLUENT_ARCH=lnamd64" to start the build. Add a "clean" to clean it.
 
 --------------------------------------------------------------------------------
 
-2.1 How to start Fluent with GUI
-- start the binary "fluent" from your simulation folder
-- set double precision, processing options (serial or parallel) and the dimension
-(for parallel select also "show more", "parallel settings", "mpi types" -> open mpi
+## 2. Installing and Starting Fluent on Ubuntu machine
 
-2.2 How to start Fluent without GUI
-- serial:   fluent 2ddp -g < steer-fluent.txt
-- parallel: fluent 2ddp -g -t4 -mpi=openmpi < steer-fluent.txt
-  (-t4 sets 4 processes for computations)
-  (steer-fluent.txt drives Fluent, can also be done by hand)
+  ### 2.1 How to install Fluent - **ANSYS V19.1 on Linux Ubuntu 16.04 LTS** 
```

Comment:
You also tried installing Fluent on Ubuntu 18.04, but it did not work. Is this correct? Can you put a comment on this here. You can also link to https://github.com/precice/fluent-adapter/issues/3